doc/classes/Array.xml

```diff
@@ -38,7 +38,6 @@
 		GD.Print(array1 + array2); // Prints [One, 2, 3, Four]
 		[/csharp]
 		[/codeblocks]
-		[b]Note:[/b] Concatenating with the [code]+=[/code] operator will create a new array, which has a cost. If you want to append another array to an existing array, [method append_array] is more efficient.
```

Comment:
Being an Espeon, I decided to look into the future and I saw that #76527 was going to be released right after my PR. So I removed this note prematurely.